common/src/main/scala/explore/GraphQLSchemas.scala

```diff
@@ -6,9 +6,11 @@ package explore
 import clue.data._
 import clue.data.syntax._
 import clue.macros.GraphQLSchema
+import eu.timepit.refined.types.string.NonEmptyString
```

Comment:
Can we make the type mapping explicit in the `Scalars` section?

Reply:
Yep.